packages/react/src/file-upload/formatFileSize.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * This is a cut down version of the filesize module from
+ * https://github.com/avoidwork/filesize.js
+ */
```

Comment:
This method has been adapted from the original by taking the default settings (we never used any custom settings) and removing any branching logic that did not apply to those defaults, and ensuring the tests continue to pass at each step.

The implementation at this stage still feels like overkill (do we really need to support `petabytes` and above?) but I wanted to pair the original down to it's minimum before removing functionality.